Ical.Net/CalendarComponents/VTimeZone.cs

```diff
@@ -83,23 +84,25 @@ public static VTimeZone FromDateTimeZone(string tzId, DateTime earliestDateTimeT
         }
         else
         {
-            // first, get the latest standard and daylight intervals, find the oldest recurring date in both, set the RRULES for it, and create a VTimeZoneInfos out of them.
-            //standard
+            // first, get the latest standard and daylight intervals, find the oldest recurring date in both,
+            // set the RRULES for it, and create a VTimeZoneInfos out of them.
+
+            // Standard
             var standardIntervals = intervals.Where(x => x.Savings.ToTimeSpan() == new TimeSpan(0)).ToList();
-            var latestStandardInterval = standardIntervals.OrderByDescending(x => x.Start).FirstOrDefault();
+            var latestStandardInterval = standardIntervals.OrderByDescending(x => x.Start).First();
```

Comment:
standardIntervals is not guaranteed to be non-empty here, so `.First()` might throw.

Reply:
Right, then could just return early with the current `vTimeZone`